### ISSUE TITLE
Unable to commit against JDBC Connection; nested exception is org.hibernate.TransactionException: Unable to commit against JDBC Connection

### ISSUE THREAD

**Issue by donggak** [awslabs/aws-mysql-jdbc#205]
**Describe the issue**
A clear and concise description of what the issue is.

**Driver Version?** 
aws-mysql-jdbc (version 1.0.0)
**Java Version?**

**OS Version?**

**MySQL Version?**
8.0.mysql_aurora.3.01.0
**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**
And what actually happens

**Logs**
If possible MySQL logs surrounding the occurrence of the issue

**Comment by donggak** [comment]
Customers say that an error occurs after changing aws-mysql-jdbc (version 1.0.0) in mariadb-java-client (version 2.3.0).
Why does this error occur?
Please let me know if you need more for analysis
Thank you.

**Comment by sergiyv-bitquill** [comment]
Hello @donggak 

Thank you for reaching out! According to the log file you provided there was a communication failure while attempting to commit a transaction. Such failure was processed by hibernate into a transaction exception.

I'd recommend to configure driver logging to get more details and be able to troubleshot the issue. Check more details about configuring logging at https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-debugging-profiling.html

**Comment by donggak** [comment]
Hello @sergiyv-bitquill
Thank you for checking and response.

I have sent an update to my customer, and I will update it here again.

**Comment by donggak** [comment]
Hello @sergiyv-bitquill
A new error occurred after the customer applied the Github guide.

2022-05-17 04:20:21 [] [WARN ]: aurora - Connection software.aws.rds.jdbc.mysql.shading.com.mysql.cj.jdbc.ConnectionImpl@2cc91d87 marked as broken because of SQLSTATE(08S01), ErrorCode(0)
software.aws.rds.jdbc.mysql.shading.com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

I have attached the error log.  (Communications+link+failure.20220517.txt)

I also looked at the MySQL Error log and observed the 'Got an error reading communication packets' message, and I guided the customer to set parameters that were not set.

[1] connect_timeout : null
[2] interactive_timeout : 600
[3] wait_timeout : null
[4] max_allowed_packet : 134217728
[5] net_read_timeout : null
[6] net_write_timeout : null

But the customer replied back to me.

Question of Cx) 
Can the connection be broken due to differences in drivers? Communications link failure did not occur when using the maria db driver on the same cluster.

The customer was previously using the maria db connector, and an issue has occurred after changing to aws mysql connector according to the recommendations in the document below.

https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Connecting.html#Aurora.Connecting.AuroraMySQL.Utilities
Version 3.0.3 of the MariaDB Connector/J utility drops support for Aurora DB clusters, so we highly recommend moving to the AWS JDBC Driver for MySQL.

Thank you.
